Makefile

```diff
@@ -12,8 +12,7 @@ build:
 	go build -o ${BINARY} .
 
 release:
-	GOOS=darwin GOARCH=amd64 go build -o ./bin/${BINARY}_${VERSION}_darwin_amd64
-	GOOS=linux GOARCH=amd64 go build -o ./bin/${BINARY}_${VERSION}_linux_amd64
+	goreleaser release --skip-publish --rm-dist --release-notes <(git-chglog "{{.Tag}}")
```

Comment:
If you're trying to run `git-chglog` in a subshell don't you need the `$`, like `$(git-chglog blah)` ?

Reply:
I am wrong, this is an example of process substitution so the output appears as a file, makes sense: https://en.wikipedia.org/wiki/Process_substitution

Reply:
Can actually get rid of this since its not used for the actual release at all